gdx/src/com/badlogic/gdx/math/Vector2.java

```diff
@@ -385,6 +415,22 @@ public Vector2 rotateRad (float radians) {
 		return this;
 	}
 
+	/** Rotates the Vector2 by the given angle around reference vector, counter-clockwise assuming the y-axis points up.

```

Comment:
could you please don't change methods order, it messing the diff and history, it looks like you added a new method rotateAround instead of modifying it.

Reply:
the change of order on `rotateAround` was deliberate to try to keep the 3 versions of each method together for clarity but that's ok, I'll revert.